modules/exploits/multi/http/cmsms_object_injection_rce.rb

```diff
@@ -0,0 +1,163 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'CMS Made Simple Authenticated RCE via object injection',
+      'Description' => %q(
+        An issue was discovered in CMS Made Simple 2.2.8.
+        In the module DesignManager (in the files action.admin_bulk_css.php
+        and action.admin_bulk_template.php), with an unprivileged user
+        with Designer permission, it is possible reach an unserialize
+        call with a crafted value in the m1_allparms parameter,
+        and achieve object injection.
+
+        This module has been successfully tested on CMS Made Simple versions
+        2.2.6, 2.2.7, 2.2.8, 2.2.9 and 2.2.9.1.
+      ),
+      'Author' => [
+        'Daniele Scanu', # Discovered and exploit. twitter.com/sk4pwn
```

Comment:
It would be better if give Author Name like this below

`'Daniele Scanu <danielescanu20[at]gmail.com>'`